operations/configuration.md

```diff
@@ -10,174 +11,186 @@ Returns the enterprise configuration. For single-enterprise Access Tokens, this
+| ~~`Address`~~ | ~~[Address](configuration.md#address)~~ | ~~required~~ | **Deprecated!** Use `AddressId` instead.|
+
+#### Pricing
+
+* `Gross` - The enterprise shows amount with gross prices.
+* `Net` - The enterprise shows amount with net prices.
 
 #### Address
 
 | Property | Type | Contract | Description |
-| :--- | :--- | :--- | :--- |
+| :-- | :-- | :-- | :-- |
 | `Id` | string | required | Unique identifier of the address. |
 | `Line1` | string | optional | First line of the address. |
 | `Line2` | string | optional | Second line of the address. |
 | `City` | string | optional | The city. |
 | `PostalCode` | string | optional | Postal code. |
-| `CountryCode` | string | optional | ISO 3166-1 code of the [Country](countries.md#country). |
+| `CountryCode` | string | optional | ISO 3166-1 code of the `Country`. |
 | `CountrySubdivisionCode` | string | optional | ISO 3166-2 code of the administrative division, e.g. `DE-BW`. |
 | `Latitude` | number | optional | The latitude. |
 | `Longitude` | number | optional | The longitude. |
 
 #### Accepted currency
 
 | Property | Type | Contract | Description |
-| :--- | :--- | :--- | :--- |
-| `Currency` | string | required | ISO-4217 code of the [Currency](currencies.md#currency). |
-| `IsDefault` | bool | required | Whether the currency is a default accounting currency. |
-| `IsEnabled` | bool | required | Whether the currency is enabled for usage. |
+| :-- | :-- | :-- | :-- |
+| `Currency` | string | required | ISO-4217 code of the `Currency`. |
+| `IsDefault` | boolean | required | Whether the currency is a default accounting currency. |
+| `IsEnabled` | boolean | required | Whether the currency is enabled for usage. |
 
-#### Pricing
+#### Accounting configuration
 
-* `Gross` - the enterprise shows amount with gross prices.
-* `Net` - the enterprise shows amount with net prices.
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `AdditionalTaxIdentifier` | string | optional, max length 50 characters | Organization number. |
+| `CompanyName` | string | optional, max length 100 characters | Legal name of the company. |
+| `BankAccountNumber` | string | optional, max length 50 characters | Bank account number. |
+| `BankName` | string | optional, max length 100 characters | Name of the bank. |
+| `Iban` | string | optional, max length 40 characters | International Bank Account Number. |
+| `Bic` | string | optional, max length 11 characters | Business Identification Code. |
+| `SurchargeConfiguration` | [Surcharging fees configuration](configuration.md#surcharging-fees-configuration) | required | Configuration for surcharging fees. |
 
-#### Payment card storage
+#### Surcharging fees configuration
 
 | Property | Type | Contract | Description |
-| :--- | :--- | :--- | :--- |
-| `PublicKey` | string | required | Key for accessing PCI proxy storage. |
+| :-- | :-- | :-- | :-- |
+| `SurchargeFees` | object | required | Surcharging fees per payment method. |
```

Comment:
Can we specify here it's Dictionary of decimals? We don't have it yet (we have dictionary of integrets) but would be nice to address it.

Reply:
Agreed, I defined it as "dictionary of numbers" (since there's really no such thing as decimal in JSON) and renamed the existing "dictionary of integers" to "dictionary of integer arrays".